app/languages/en/index.js

```diff
@@ -14,6 +14,7 @@ import SelectSpecies from './user/selectSpecies.json';
 import MapMarking from './user/mapMarking.json';
 import TreeInventory from './user/treeInventory.json';
 import Review from './user/treeReview.json';
+import SpeciesSyncError from './user/speciesSyncError';
```

Comment:
Adding new language files always has to be done in all supported languages in the `index.js` file in the language folder.